packages/mail/src/mail.spec.js

```diff
@@ -36,6 +36,8 @@ describe('sgMail.send()', () => {
     html: '<p>Hello HTML world!</p>',
   };
 
+  const cb = {};
```

Comment:
Can probably remove this and just pass the object directly in the test to avoid clutter.

Reply:
Removed